splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/export/ExportParams.java

```diff
@@ -117,8 +117,27 @@ private void setFormat(String format) throws StandardException {
         this.format = format;
     }
 
-    private void setCompression(Boolean compression) throws StandardException {
-        this.compression = compression;
+    private void setCompression(String compression) throws StandardException {
```

Comment:
This breaks the current uses of Export(), we should handle true/false too

Reply:
We cannot keep the current export() API and introduce different compression options while still keeping true/false.